k8s/upgrades/0.8.2-0.9.0/cstor/cstor-pool-update-082-090.yaml

```diff
@@ -164,6 +200,9 @@ spec:
     {{- jsonpath .JsonResult "{.spec.template.spec.containers[?(@.name=='cstor-pool-mgmt')].image}" | trim | saveAs "poolDeployment.cstorpoolmgmtimage" .TaskResult | noop -}}
     {{- jsonpath .JsonResult "{.metadata.labels.openebs\\.io/version}" | trim | saveAs "poolDeployment.version" .TaskResult | noop -}}
 
+    {{- jsonpath .JsonResult "{.spec.replicas}" | trim | saveAs "poolDeployment.replicaCount" .TaskResult | noop -}}
+    {{- .TaskResult.poolDeployment.replicaCount | notFoundErr "unknown no.of replicas in pool deployment" | saveIf "poolDeployment.notFoundErr" .TaskResult | noop -}}
```

Comment:
message can be
`No replica count set for pool deployment`
This can be a verifyError.

Reply:
Updated the error message